driver/src/main/com/mongodb/reactivestreams/client/ChangeStreamPublisher.java

```diff
@@ -49,6 +50,20 @@
      */
     ChangeStreamPublisher<TResult> resumeAfter(BsonDocument resumeToken);
 
+    /**
+     * The change stream will only provides changes that occurred after the specified timestamp.
```

Comment:
Minor change to: The change stream will only provide changes that occurred at or after the specified timestamp.